app/views/users/emails/show.html.erb

```diff
@@ -14,7 +14,7 @@
   <% end %>
 </div>
 
-<%= render 'shared/cancel', link: decorated_sp_session.cancel_link_url %>
+<%= render 'shared/cancel', link: @cancel_link_url %>
```

Comment:
The designs seem to show the text as being "Back" in this workflow, but AFAICT this would be showing the link text as "Cancel".

Reply:
I see. Good catch!